tock/tock/templates/employees/user_form.html

```diff
@@ -4,7 +4,7 @@
 {% endblock %}
 
 {% block content %}
-<h1>Let's Tock about {{ user.get_full_name }}!</h1>
+<h1>Let's Tock about {{ display_name }}!</h1>
```

Comment:
I haven't looked at the view to confirm, but it looks like this page is passed a `user` object, so I don't think a change is needed and `user.get_full_name` should still be correct.

Reply:
Ah, we're passing display_name to the form?

Reply:
i think passing the `display_name` through will be the most consistent experience -- though happy to change it if you disagree strongly!